symplyphysics/laws/electricity/circuits/characteristic_resistance_of_rectangular_waveguide_for_e_mn_waves.py

```diff
@@ -0,0 +1,41 @@
+from sympy import Eq, solve, sqrt
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output,)
+
+## Description
+## A rectangular waveguide is a rectangular metal waveguide capable of supporting waves propagating along it.
+## There is a critical wavelength. Signals with a wavelength greater than the critical one are attenuated and
+## do not propagate in the waveguide.
+## The characteristic resistance of a wave is a value determined by the ratio of the transverse component
+## of the electric field strength to the transverse component of the magnetic field strength of a traveling wave.
+
+## Law is: Z= Z0 * sqrt(1 - (L / L1)^2), where
+## Z - characteristic resistance of rectangular waveguide,
+## Z0 - characteristic resistance of medium,
+## L - wavelength,
+## L1 - critical wavelength.
```

Comment:
Add wave mode to conditions.

Reply:
> Add wave mode to conditions.

This formula is for any mode (any pair of indices m and n). The critical wavelength depends on the indices.

Reply:
Yes, but TM wave should be in conditions.